# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mediasink/AudioSinkWrapper.h
# Commit: 16633fa96586
# Full Hash: 16633fa96586cb01af6dbfb03d062f1b32691ae9
# Author: Paul Adenot <paul@paul.cx>
# Date: 2022-05-24 21:44:48
# Regressor Bug: 1743834
# File Overlap Count: 1
# Description:
#   Bug 1743834 - Split initialization and start in AudioSinkWrapper. r=alwu
#   
#   No change in functionality, this is to prepare for the next patch that will do
#   the initialization asynchronously because it's expensing and blocking.
#   
# ==============================================================================

diff -r 505d7df63978 -r 16633fa96586 dom/media/mediasink/AudioSinkWrapper.h
--- a/dom/media/mediasink/AudioSinkWrapper.h	Tue May 24 13:09:09 2022 +0000
+++ b/dom/media/mediasink/AudioSinkWrapper.h	Tue May 24 13:09:09 2022 +0000
@@ -30,7 +30,7 @@
   class Creator {
    public:
     virtual ~Creator() = default;
-    virtual AudioSink* Create(const media::TimeUnit& aStartTime) = 0;
+    virtual AudioSink* Create() = 0;
   };
 
   // Wrap around a function object which creates AudioSinks.
@@ -38,9 +38,7 @@
   class CreatorImpl : public Creator {
    public:
     explicit CreatorImpl(const Function& aFunc) : mFunction(aFunc) {}
-    AudioSink* Create(const media::TimeUnit& aStartTime) override {
-      return mFunction(aStartTime);
-    }
+    AudioSink* Create() override { return mFunction(); }
 
    private:
     Function mFunction;
